plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -158,10 +156,7 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.assertj.core.api.Assertions.offset;
 import static org.junit.jupiter.api.Assumptions.abort;
-import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
 
-@Isolated // TODO remove
```

Comment:
nit: does this need stress testing on CI?

Reply:
i don't see anything special about this class.

Reply:
the isolated was removed but I have no idea why this is safe.

Reply:
also iceberg CI is red.

Reply:
good point. but https://github.com/trinodb/trino/issues/21736 is also a known flake, so i don't feel guilty.

Reply:
> the isolated was removed but I have no idea why this is safe.

i have no idea why it was added in the first place